### ISSUE TITLE
Campaign: Polling Stations [Wahllokale]

### ISSUE THREAD

**Issue by iantrieschmann** [verdigado/gruene_app#220]
**Wenn Nutzer:innen versuchen, im Umkreis von 20m um ein Wahllokal ein Plakat zu registrieren, dann wird eine Fehlermeldung ausgegeben: "Du befindest dich zu dicht an einem Wahllokal. Im Umkreis von 20m um Wahllokale dürfen keine Wahlplakate hängen."**

Anmerkungen:
- Es ist bisher unklar, ob es überhaupt eine Datenbank zu den Wahllokalen gibt. Ohne diese können die Wahllokalen nicht vordefiniert und auf der Karte angezeigt werden. Sollte es eine Datenbank für die Wahllokalen geben, wäre es von Vorteil für die Wahlkämpfer*innen, wenn es in der App eine entsprechende Warnung gäbe. Dieses Teilfeature ist aber ein nice-to-have.
- Eine Alternative wäre es, Wahllokale als erstellbares und verschiebbares Element (analog zu Plakaten) auf der Karte erstellen und entfernen zu lassen, damit die Wahlkämpfenden diese selber eintragen und nach der Wahl löschen können. Eine offene Frage dabei wäre: ist diese Art von Datenspeicherung und -verarbeitung zulässig.

![grafik](https://github.com/user-attachments/assets/9cc7a4f6-99be-46c1-ab43-774c2286a1cf)

Prototype: https://www.figma.com/proto/7wE0Iveas3GGb8u4oL0714/App-Design-Mitglieder-App-(1)-(Copy)?node-id=3112-15401&t=b8swD5IOAFfZ0OSo-1&starting-point-node-id=2425%3A17429

**Comment by nscharioth** [comment]
Pollion reference: https://github.com/pollion/sunflower-app/issues/80

**Comment by nscharioth** [comment]
Please update initial text provided by Ian. @leagerndt @JuliaFreitagBGSt

**Comment by iantrieschmann** [comment]
Concept requires further refinement (see related discussion in #274). Due to this and the priority of other features for the federal elections in 2025, polling stations are to be parked in the backlog until further notice / capacities become available.
@nscharioth fyi

**Comment by georgloesel** [comment]
just a comment how to get the location of the polling stations
OpenData portals all over Germany provide this information. 

some examples: 

- https://wahlen.wuppertal.de/bw2025/05124000/praesentation/wahlraeume.html
- https://opendata.potsdam.de/explore/dataset/wahllokale-europa-und-kommunalwahl-2024-landeshauptstadt-potsdam/table/?flg=de-de&sort=wahllokale
- https://opendata.ruhr/dataset/wahllokale/resource/387a5d50-2218-4c37-8452-9538d46ef6f9
- https://daten.berlin.de/datensaetze/geometrien-der-wahllokale-fur-die-wahl-zum-21-deutschen-bundestag-in-berlin

Unfortunately the structure is not defined so that an automatic API call to the local opendata instance might be impossible. 

probably an import of points and manual managmeent via atlas might be an altrnative.

**Comment by nscharioth** [comment]
> just a comment how to get the location of the polling stations OpenData portals all over Germany provide this information.
> 
> some examples:
> 
> * https://wahlen.wuppertal.de/bw2025/05124000/praesentation/wahlraeume.html
> * https://opendata.potsdam.de/explore/dataset/wahllokale-europa-und-kommunalwahl-2024-landeshauptstadt-potsdam/table/?flg=de-de&sort=wahllokale
> * https://opendata.ruhr/dataset/wahllokale/resource/387a5d50-2218-4c37-8452-9538d46ef6f9
> * https://daten.berlin.de/datensaetze/geometrien-der-wahllokale-fur-die-wahl-zum-21-deutschen-bundestag-in-berlin
> 
> Unfortunately the structure is not defined so that an automatic API call to the local opendata instance might be impossible.
> 
> probably an import of points and manual managmeent via atlas might be an altrnative.

Yes, the situation is similar to what we had with the polygons for the focus areas. We usually have to go to various municipalities directly. 
For the research we have the following issues to track this: https://github.com/pollion/sunflower-data/issues/7 and https://github.com/pollion/sunflower-data/issues/8.
